env0/provider.go

```diff
@@ -48,33 +48,34 @@ func Provider(version string) plugin.ProviderFunc {
-				"env0_template":                dataTemplate(),
-				"env0_templates":               dataTemplates(),
-				"env0_ssh_key":                 dataSshKey(),
-				"env0_aws_cost_credentials":    dataCredentials(AWS_COST_TYPE),
-				"env0_azure_cost_credentials":  dataCredentials(AZURE_COST_TYPE),
-				"env0_google_cost_credentials": dataCredentials(GCP_COST_TYPE),
-				"env0_aws_credentials":         dataCredentials(AWS_TYPE),
-				"env0_gcp_credentials":         dataCredentials(GCP_TYPE),
-				"env0_azure_credentials":       dataCredentials(AZURE_TYPE),
-				"env0_team":                    dataTeam(),
-				"env0_teams":                   dataTeams(),
-				"env0_environment":             dataEnvironment(),
-				"env0_workflow_triggers":       dataWorkflowTriggers(),
-				"env0_notification":            dataNotification(),
-				"env0_notifications":           dataNotifications(),
-				"env0_module":                  dataModule(),
-				"env0_git_token":               dataGitToken(),
-				"env0_api_key":                 dataApiKey(),
-				"env0_agents":                  dataAgents(),
-				"env0_user":                    dataUser(),
-				"env0_cloud_credentials":       dataCloudCredentials(),
-				"env0_source_code_variables":   dataSourceCodeVariables(),
-				"env0_ip_ranges":               dataIpRanges(),
+				"env0_organization":             dataOrganization(),
+				"env0_project":                  dataProject(),
+				"env0_project_policy":           dataPolicy(),
+				"env0_configuration_variable":   dataConfigurationVariable(),
+				"env0_template":                 dataTemplate(),
+				"env0_templates":                dataTemplates(),
+				"env0_ssh_key":                  dataSshKey(),
+				"env0_aws_cost_credentials":     dataCredentials(AWS_COST_TYPE),
+				"env0_azure_cost_credentials":   dataCredentials(AZURE_COST_TYPE),
+				"env0_google_cost_credentials":  dataCredentials(GCP_COST_TYPE),
+				"env0_aws_credentials":          dataCredentials(AWS_TYPE),
+				"env0_gcp_credentials":          dataCredentials(GCP_TYPE),
+				"env0_azure_credentials":        dataCredentials(AZURE_TYPE),
+				"env0_team":                     dataTeam(),
+				"env0_teams":                    dataTeams(),
+				"env0_environment":              dataEnvironment(),
+				"env0_workflow_triggers":        dataWorkflowTriggers(),
+				"env0_notification":             dataNotification(),
+				"env0_notifications":            dataNotifications(),
+				"env0_module":                   dataModule(),
+				"env0_git_token":                dataGitToken(),
+				"env0_api_key":                  dataApiKey(),
+				"env0_agents":                   dataAgents(),
+				"env0_user":                     dataUser(),
+				"env0_cloud_credentials":        dataCloudCredentials(),
+				"env0_source_code_variables":    dataSourceCodeVariables(),
+				"env0_ip_ranges":                dataIpRanges(),
+				"env0_self_hosted_agent_values": dataSelfHostedAgentValues(),
```

Comment:
I wonder if we should name it `env0_self_hosted_agent_values` and not `env0_agent_values` (as we already named it `env0_agents` on another data resource). WDYT?

Reply:
you prefer it to be named env0_agent_values?
I'm okay either way.

Reply:
Yes. I'd keep it consistent with the existing `env0_agents` data resource 🙏

Reply:
sure!

Reply:
@yaronya - done!
b2c3075fa2dde61bc71d36f07d1a86312f32a402